### ISSUE TITLE
Type doc mistake Enumeration type as union

### ISSUE THREAD

**Issue by wuhanyumsft** [TypeStrong/typedoc#599]
I found an issue that TypeDoc mistake Enumeration type as union type.

Here is a sample:
A property named `artifactType` is a enumeration type named `AgentArtifactType`. But TypeDoc mistake it as a union type, while display its values as the member of union type.

```
{
	"id": 127169,
	"name": "artifactType",
	"kind": 1024,
	"kindString": "Property",
	"flags": {
		"isExported": true
	},
	"sources": [
		{
			"fileName": "rmo.d.ts",
			"line": 148,
			"character": 16
		}
	],
	"type": {
		"type": "union",
		"types": [
			{
				"type": "unknown",
				"name": "AgentArtifactType.XamlBuild"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.Build"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.Jenkins"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.FileShare"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.Nuget"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.TfsOnPrem"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.GitHub"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.TFGit"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.ExternalTfsBuild"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.Custom"
			},
			{
				"type": "unknown",
				"name": "AgentArtifactType.Tfvc"
			}
		]
	}
}
```
Can we fix that?  Thanks.

**Comment by wuhanyumsft** [comment]
Is there anyone can help?

**Comment by aciccarello** [comment]
I would have expected this to be fixed with #599 though someone else seems to be having an issue with string enums. Are you using the latest version of TypeDoc?

**Comment by fabioberger** [comment]
Plus one for this issue. Just ran into it myself with string enums.

**Comment by wuhanyumsft** [comment]
I ran the result with typedoc version 0.9.0

**Comment by Gerrit0** [comment]
This appears to be fixed, I can't reproduce with 0.15.

https://typedoc.org/api/classes/logger.html#log
